### ISSUE TITLE
Classification Objectives

### ISSUE THREAD

**Issue by dhuppenkothen** [dirac-institute/classE#3]
I can think of three main objectives the light curve classification engine might have:

* classification of transients for follow-up
* classification of transients to find outliers
* classification of transients to do population studies

All three require slightly different approaches, I think. For example, for transient follow-up, one might want to minimize the false-positive rate on certain transients, but this unlikely a good objective for basing population studies off a classifier. 
The solution might be to build a system that makes acceptable trade-offs, but that requires defining what those trade-offs are and what our minimum requirements on each objective is. 

Are there other objectives I've not thought of?

**Comment by MelissaGraham** [comment]
Usually if I am doing anything with light curves, it is deriving physical parameters for individual supernovae (e.g., distances, nickel mass, host extinction), which can be a different approach again from the three listed. In @connolly 's original email he mentioned light curve fitting and analysis in addition to classification, but this GitHub repo is titled "classE" and this particular issue titled "Classification Objectives", so perhaps this fourth kind of objective is beyond the scope, but I thought I'd mention it here anyway.

**Comment by dhuppenkothen** [comment]
Good point @MelissaGraham! This wasn't meant to be comprehensive, more a starting point for brainstorming. Are you mainly interested in the physical parameters of individual supernovae for their own sake, or if you had 20 supernovae observed with ZTF, would you want to infer something about the distribution of, say, nickel mass across all supernovae?

In my head, I would see the analysis of individual light curves as the step following the classification: presumably, different models are used for different types of transients, and so any analysis would assume a successful classification already done? 

I think we'll have to think hard about how biases in our classification affect the subsequent analysis, especially for population inference, and in return our classification algorithms should be guided by what follow-up analysis we are interested in doing. So I'd say this absolutely belongs here, too.
